docs/serving/encryption/encryption-overview.md

```diff
@@ -0,0 +1,50 @@
+# Serving Encryption Overview
+
+{% include "encryption-notice.md" %}
+
+There are three parts to Knative Serving encryption:
+
+1. HTTPS on the ingress layer _external_ to the cluster (cluster external domain, like `myapp-<namespace>.example.com`).
+1. HTTPS on the ingress layer _internal_ to the cluster (cluster local domains, like `myapp.<namespace>.svc.cluster.local`).
+1. HTTPS between Knative internal components (`ingress-controller`, `activator`, `queue-proxy`).
+
+![Overview of Knative encryption](./encryption-overview.drawio.svg)
+
+!!! note
+    Currently, all control-plane traffic (including Kubernetes PreStopHooks and metadata like metrics) is not encrypted.
+
+## The parts in detail
+The different parts are independent of each other and (can) use different Certificate Authorities to sign certificates.
```

Comment:
From a user perspective what is the value of allowing part 2, without part 3?
What setup do we recommend here?

Reply:
I don't think it is really. It just gives the user the option to do the transition in steps, e.g. maybe they already have 1, they could start with 3 and add 2 once they have the CA stuff figured out. There is not really a recommendation, this highly depends on the environment and the requirements of a company.